include/af/device.h

```diff
@@ -575,6 +575,46 @@ extern "C" {
     */
     AFAPI af_err af_get_device_ptr(void **ptr, const af_array arr);
 
+#if AF_API_VERSION >= 38
+    /**
+       Sets the path where the kernels generated at runtime will be cached
+
+       Sets the path where the kernels generated at runtime will be stored to
+       cache for later use. The files in this directory can be safely deleted.
+       The default location for these kernels is in $HOME/.arrayfire on Unix
+       systems and in the ArrayFire temp directory on Windows.
```

Comment:
I think we can better illustrate the lookup logic as an ordered list of entries such as below. We can then hyperlink this order of lookup from environment variables page documentation also.
1. Environment variable path if writable
2. List of predetermined paths as per the operation system.
    - Windows
        1. ArrayFire application Temp folder 
    - Linux
       1. Under `.arrayfire` folder of user home directory.
       2. `/tmp/arrayfire`